### PR TITLE
fix(cli): Update amount denominations in mint/burn to be IRON

### PR DESCRIPTION
**Pull request by rohanjadvani**
## Summary

Make amounts consistent with send

## Testing Plan

Manual test

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

**Comment by mat-if** [comment]
I think we're double encoding a value or something on this display:

image

**Comment by rohanjadvani** [comment]
Should be fixed now.

image

File: ironfish/src/rpc/routes/wallet/burnAsset.ts [hunk 1]
```diff
@@ -73,6 +75,7 @@ router.register<typeof BurnAssetRequestSchema, BurnAssetResponse>(
     request.end({
       assetId: burn.assetId.toString('hex'),
       hash: transaction.hash().toString('hex'),
+      value: burn.value.toString(),
```
Comment (NullSoldier) [inline ironfish/src/rpc/routes/wallet/burnAsset.ts:78]: CurrencyUtils.encode(burn.value)

Comment (NullSoldier) [inline ironfish/src/rpc/routes/wallet/burnAsset.ts:78]:
```suggestion
      value: CurrencyUtils.encode(burn.value),
```

**Review by NullSoldier — APPROVED** [review]

**Pull request merged**